src/seeders/20201124160449-users.js

```diff
@@ -167,6 +176,7 @@ const users = [
   {
     id: 7,
     email: 'josh@adhocteam.us',
+    hsesUsername: 'josh@adhocteam.us',
     role: 'Grants Specialist',
     name: 'Josh',
     hsesUserId: '50154',
```

Comment:
Should we add other devs to the seeders too? Apparently my hsesUserId is `50387`